docs/cfc.md

```diff
@@ -6,5 +6,7 @@ To use, run the pipeline with `-profile cfc`. This will download and launch the
 
 Before running the pipeline you will need to install Nextflow on the CFC cluster. You can do this by following the instructions [here](https://www.nextflow.io/).
 
+This config includes the necessary parameters for estimating the CO2 footprint via the [nf-co2footprint plugin](https://github.com/nextflow-io/nf-co2footprint) of any pipeline run on the cluster. The parameters are the cluster-specific power usage effectiveness (PUE)  of `1.3` and the location of the cluster in Germany(`DE`).
```

Comment:
```suggestion
This config includes the cluster-specific parameters for estimating the CO2 footprint via the [nf-co2footprint plugin](https://github.com/nextflow-io/nf-co2footprint) of any pipeline run on the cluster. The parameters include the cluster-specific power usage effectiveness (PUE) and the location of the cluster in Germany(`DE`).
```

Reply:
pretty sure otherwise we will forget updating this in the future in case of changes